samples/file-encryptor/enclave/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@
 # Use the edger8r to generate C bindings from the EDL file.
 add_custom_command(OUTPUT fileencryptor_t.h fileencryptor_t.c fileencryptor_args.h
   DEPENDS ${CMAKE_SOURCE_DIR}/fileencryptor.edl
-  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/fileencryptor.edl)
+  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/fileencryptor.edl
+                                 --search-path ${OE_INCLUDEDIR}
+                                 --search-path ${OE_INCLUDEDIR}/openenclave/edl/sgx)
```

Comment:
Hernan mentioned that he would expect the file-encryptor sample to work for OP-TEE, but line 9 seems to add SGX-specific stuff into the makefile without being inside an SGX condition.

Reply:
I asked previously but didn't get an answer. Doesn't line 21 of `samples/file-encryptor/CMakeLists.txt` already mean the sample can't be built for OP-TEE?

If OP-TEE wasn't already supported, I think adding an OP-TEE build flag for these samples is out of scope for this PR

Reply:
Sorry. I see what you mean, you are right. I was thinking that in terms of the code it should work, but it's true that the build steps currently only generate properly signed SGX enclaves, and not OP-TEE TAs.

I had brought up the idea at one point to export the `add_enclave` function from the build system to consumers (i.e., when consuming the CMake exports from a binary release), so this signing logic wouldn't have to be copied around by users.

I do agree that solving that problem is out of scope for this PR, and so is duplicating the appropriate signature commands for OP-TEE; they are far less trivial than for SGX.

Reply:
Ok sounds good. is there an issue to track enabling samples on OP-TEE? If not I can open one.

Reply:
I don't there is one.

Reply:
Opened https://github.com/openenclave/openenclave/issues/2997

Thanks!